scripts/Diffraction/isis_powder/polaris.py

```diff
@@ -50,11 +50,7 @@ def create_vanadium(self, **kwargs):
         return vanadium_d
 
     def create_total_scattering_pdf(self, **kwargs):
-        if 'q_lims' not in kwargs:
-            kwargs['q_lims'] = None
-        if 'output_binning' not in kwargs:
-            kwargs['output_binning'] = None
-        if 'pdf_type' not in kwargs or not kwargs['pdf_type'] in ['G(r)', 'g(r)', 'RDF(r)']:
+        if 'pdf_type' not in kwargs or kwargs['pdf_type'] not in ['G(r)', 'g(r)', 'RDF(r)']:
```

Comment:
Is this just a style change rather than a functional change? I think "not x in [list]" is the same as "x not in [list]

Reply:
It is style, I wanted the two statements to match